jupyter_notebook/notebookapp.py

```diff
@@ -203,6 +241,7 @@ def init_settings(self, ipython_app, kernel_manager, contents_manager,
             config=ipython_app.config,
             jinja2_env=env,
             terminals_available=False,  # Set later if terminals are available
+            log=log,
```

Comment:
I though it looked like settings was being used as a generic budle of things, but I must have been imagining it.  I'll set it on self.